### ISSUE TITLE
Add links to test results in the OSD integ test notification

### ISSUE THREAD

**Issue by tianleh** [opensearch-project/opensearch-build#1728]
### Is your feature request related to a problem? Please describe

Currently only the Jenkins job link is posted in the OSD integ test notification. This is not convenient to look at where the failures are. Since the test result is uploaded under CloudFront, we can include the ci endpoint for test results.

E.g

https://ci.opensearch.org/ci/dbc/integ-test-opensearch-dashboards/1.3.0/1484/linux/x64/test-results/b94f426463b24046a2ed33c5be290508/integ-test/functionalTestDashboards/without-security/test-results/stdout.txt

At minimum, we shall include the url to stdout.txt for `without-security` and `with-security`. We can also consider include links to the video and screenshots.

### Describe the solution you'd like

At minimum, we shall include the url to stdout.txt for `without-security` and `with-security`. We can also consider include links to the video and screenshots.

**Comment by tianleh** [comment]
Currently the url has a random hash which makes it difficult to share. We can pass a test run ID to https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch-dashboards/integ-test.jenkinsfile#L68

Then the location of test file will use the input instead of random value https://github.com/opensearch-project/opensearch-build/blob/main/src/test_workflow/test_recorder/test_recorder.py#L21

Then we will be able to construct the test result url.

**Comment by tianleh** [comment]
cc @kavilla

**Comment by bbarani** [comment]
@tianleh Can you update the current status?

**Comment by tianleh** [comment]
> @tianleh Can you update the current status?

The issue has a pre-request PR https://github.com/opensearch-project/opensearch-build/pull/2028/files which was waiting for post 2.0 release. I will bring the related PR to sync up with latest main branch. After that, the issue just needs to include the test result link in the message notification.

**Comment by bbarani** [comment]
@tianleh Can you update the current status?

**Comment by gaiksaya** [comment]
Thanks to @zelinh we recently introduced reporting workflow https://github.com/opensearch-project/opensearch-build/issues/1274 that generated an overview of all the tests and includes sub-reports that contains links to all test files including videos, stdout, stderr, etc. 
Example: https://github.com/opensearch-project/dashboards-observability/issues/1031#issue-1903802966

Closing this issue. Thanks!
